packages/web-app-files/src/mixins/spaces/actions/disable.js

```diff
@@ -5,17 +5,21 @@ export default {
     $_disable_items() {
       return [
         {
-          name: 'delete',
+          name: 'disable',
           icon: 'forbid-2',
           label: () => {
             return this.$gettext('Disable')
           },
-          handler: this.$_disable_showModal,
-          isEnabled: (space) => {
-            return !(space.root?.deleted?.state === 'trashed')
+          handler: this.$_disable_trigger,
+          isEnabled: ({ spaces }) => {
+            if (spaces.length === 0) {
```

Comment:
you can check for `spaces.length !== 1` here since you only allow the action for a single space as of now.